### ISSUE TITLE
codeclimate works incorrectly on servers/lib directory

### ISSUE THREAD

**Issue by prasadtalasila** [INTO-CPS-Association/DTaaS#61]
Codeclimate does not use the top-level `codeclimate.yml` configuration for `servers/lib` directory.

**Comment by prasadtalasila** [comment]
Does this have to do with issue #36?

**Comment by prasadtalasila** [comment]
> Does this have to do with issue #36?

no

**Comment by prasadtalasila** [comment]
This is because `.codeclimate.yml` uses the eslint config in the following code fragment:

```
...
  eslint:
    enabled: true
    channel: "eslint-8"
    config:
      config: client/.eslintrc
...
```

The `client/` and `servers/lib` have different `.eslintrc` files. The codeclimate has to use both configs, perhaps path-dependent configs.

**Comment by prasadtalasila** [comment]
this problem is resolved in PR #92
